multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -1067,3 +1063,8 @@ def create_transaction_for_registering_dynamic_and_setting_roles(self,
             add_data_movement_gas=True,
             data_parts=parts
         ).build()
+
+    def _bool_to_typed_string(self, value: bool) -> StringValue:
+        if value:
+            return StringValue("true")
```

Comment:
The class has the two pre-saved members: `_true_as_typed_value` and `_false_as_typed_value`. Can be used here?

Reply:
There's no need to use those, I've deleted them.